samples/CallingStateful/src/App.tsx

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useCallback, useEffect, useState } from 'react';
+import './App.css';
+import { CommunicationUserIdentifier } from '@azure/communication-common';
+import {
+  CallCommon,
+  IncomingCall,
+  TeamsIncomingCall,
+  LocalVideoStream,
+  IncomingCallCommon
+} from '@azure/communication-calling';
+/* @conditional-compile-remove(one-to-n-calling) */
+import { IncomingCallEvent, TeamsIncomingCallEvent } from '@azure/communication-calling';
+import {
+  DEFAULT_COMPONENT_ICONS,
+  FluentThemeProvider,
+  StatefulCallClient,
+  CallClientState
+} from '@azure/communication-react';
+/* @conditional-compile-remove(one-to-n-calling) */
+import { DeclarativeCallAgent, DeclarativeTeamsCallAgent } from '@azure/communication-react';
+import { Stack, Text, initializeIcons, registerIcons } from '@fluentui/react';
+import heroSVG from './assets/hero.svg';
+import { LoginScreen } from './views/Login';
+import { CallScreen } from './views/CallScreen';
+import { IncomingCallManager } from './components/IncomingCallManager';
+initializeIcons();
+registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
+
+function App(): JSX.Element {
+  const imageProps = { src: heroSVG.toString() };
+  const [userIdentifier, setUserIdentifier] = useState<CommunicationUserIdentifier>();
+  const [teamsIdentifier, setTeamsIdentifier] = useState<string>();
+
+  const [userCredentialFetchError, setUserCredentialFetchError] = useState<boolean>(false);
+
+  const [statefulCallClient, setStatefulCallClient] = useState<StatefulCallClient>();
+  /* @conditional-compile-remove(one-to-n-calling) */
+  const [callAgent, setCallAgent] = useState<DeclarativeCallAgent | DeclarativeTeamsCallAgent>();
+  const [call, setCall] = useState<CallCommon>();
+  const [incomingCalls, setIncomingCalls] = useState<readonly IncomingCall[] | readonly TeamsIncomingCall[]>([]);
```

Comment:
Would this App be similar to our stateful quickstart in stable with no incoming calls? And it would have incoming once you GA incoming calls right?